tests/playwright/src/model/ai-lab-base-page.ts

```diff
@@ -0,0 +1,37 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+
+export class AILabBasePage {
+  readonly page: Page;
+  readonly webview: Page;
+  heading: Locator;
+
+  constructor(page: Page, webview: Page) {
+    this.page = page;
+    this.webview = webview;
+    this.heading = webview.getByRole('heading', { name: 'Welcome to Podman AI Lab' });
```

Comment:
Heading is always visible?

Reply:
probably leftover. remove it please.

Reply:
yes, except for one page, but it's just a locator, so shouldn't cause trouble until invoked

Reply:
And you have it there, so remove it since if invoked, it will fail.

Reply:
refactored